pages/_app.js

```diff
@@ -16,6 +27,26 @@ function MyApp({ Component, pageProps }) {
     );
   }
 
+  // add a memory
+  function handleAddCreateCard(event) {
+    event.preventDefault();
+    const date = event.target.date.value;
+    const memory = event.target.memory.value;
+    const isFavorite = event.target.isFavorite.checked;
+
+    const newEntry = {
+      id: nanoid(),
+      name: memory,
+      date: date,
```

Comment:
wird bei neuen Erinnerungen in einem anderen Format angezeigt, als deine initialEvents

Reply:
hängt mit dem Problem zusammen, wie oben beschrieben